playbooks/prepare_host.yaml

```diff
@@ -2,20 +2,42 @@
 - hosts: standalone
   become: true
   become_user: root
-  gather_facts: false
+  gather_facts: true
   vars_files: vars/defaults.yaml
 
   tasks:
 
-  - name: install rhos-release
-    yum:
-      state: installed
-      name:
-      - http://download.devel.redhat.com/rcm-guest/puddles/OpenStack/rhos-release/rhos-release-latest.noarch.rpm
-      disable_gpg_check: True
+  - name: Prepare host on RHEL system
+    when:
+      - ansible_facts.distribution == 'RedHat'
+    block:
+      - name: install rhos-release
+        yum:
+          state: installed
+          name:
+          - http://download.devel.redhat.com/rcm-guest/puddles/OpenStack/rhos-release/rhos-release-latest.noarch.rpm
+          disable_gpg_check: True
+
+      - name: Configure rhos release "{{ rhos_release }}"
+        shell: rhos-release "{{ rhos_release }}"
+
+      - name: Fetch the Red Hat root certificate
```

Comment:
This is likely something we want regardless of whether we're deploying OSP or RDO. IIRC we need this to pull from internal registries when installing openshift.